doc/UserGuide.md

```diff
@@ -44,6 +44,9 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > be seen using the `viewall` command.
 > 
 > Persons can have any number of tags (including 0)
+> When adding people, the input will be standardised as such: Names will follow the format of first letter upper-case, rest lower-case.
+> When adding email, the input will be standardised to all lower-case.
+> When adding address, the input will be standardised as such: Addresses will follow the format of first letter upper-case, rest lower-case.
```

Comment:
Not sure about how appropriate this is for addresses since there are building names in caps (ie. TIC Tech Centre)